docs/Maintainer-Guidelines.md

```diff
@@ -9,6 +9,10 @@ Maybe you were looking for the [Formula Cookbook](Formula-Cookbook.md)?
 
 This document is current practice. If you wish to change or discuss any of the below: open a PR to suggest a change.
 
+## Mission
+
+Homebrew aims to be the missing package manager for macOS. Its primary goal is to be useful. It should be as useful as possible to as many people as possible while remaining maintainable by a small group of volunteers to a commercially high standard. Where possible and sensible it should seek to features of macOS to blend in with the macOS and Apple ecosystem.
```

Comment:
```suggestion
Homebrew aims to be the missing package manager for macOS. Its primary goal is to be useful. It should be as useful as possible to as many people as possible while remaining maintainable by a small group of volunteers to a high standard. Where possible and sensible it should seek to use features of macOS to blend in with the macOS ecosystem.
```

Reply:
LGTM

Reply:
The "commercially" part is important; it's what defines limiting what we "officially" support to stuff that actually works. The Apple ecosystem part is also important as we want to fit in with the wider Apple ecosystem beyond just macOS.

Reply:
> it's what defines limiting what we "officially" support to stuff that actually works

Even after this clarification, I'm not sure what you mean by "commercially" in this context. I'm not trying to be a pedantic pain; I genuinely don't understand. By "commercially" you mean to say that "we officially support stuff that actually works", as opposed to what? To quote Princess Bride, "I do not think it means what you think it means." =p

Reply:
Does "professionally" have a similar meaning to "commercially" in this context? ie "a professionally high standard"

Reply:
No, these are good questions and I'm glad they are asked. Many open source projects (including Homebrew in the past) have a "well X kinda works but I wouldn't recommend it" attitude to various features/repositories/etc. That does not fly for commercial organisations and should not fly for us. In short, our benchmark should be software that is being sold for money rather than software that is available free of charge. Similarly, we should not see ourselves as a hobby or club but a organisation that behaves professionally and treats our fellow maintainers as we would coworkers.

Rambly, I know, but hopefully that helps a bit?

Reply:
I understand now. Thanks for the clarification, Mike. I like Steven's suggestion. How about
```
… to a professional standard of quality.
```